core/src/main/scala/org/genivi/sota/core/AutoInstallResource.scala

```diff
@@ -89,7 +61,7 @@ class AutoInstallResource
             removeAll(ns, pkgName)
           }
         } ~
-        devicePathExtractor(ns) { devUuid =>
+        devicePathExtractor(ns)(ec) { devUuid =>
```

Comment:
I don't really know a simpler way to do it. Also the error message was pretty hard to understand, so I was happy to pass it explicitely.